tests/node_suspension.py

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+import os
+import getpass
+import time
+import logging
+import multiprocessing
+import shutil
+from random import seed
+import infra.ccf
+import infra.proc
+import infra.jsonrpc
+import infra.notification
+import infra.net
+import e2e_args
+from threading import Timer
+import random
+
+from loguru import logger as LOG
+
+# 256 is the number of messages PBFT keeps in memory before needing to replay the ledger
+# by setting 200 pure requests we will surpass 256 messages being passed throught the protocol
```

Comment:
I don't understand this comment. Are we saying that sending 200 requests will cause more than 256 messages to be sent in PBFT?

Reply:
I will rephrase, but the comment is essentially saying that with 200 client requests along with all the join and election RPCs, we will have more than 256 messages passing through the consensus protocol in total. It might be clearer if we just set this to 256 (which is the "checkpoint interval" in pbft), even though it will make the test slightly longer (~1min to ~1m10s)